### ISSUE TITLE
[RFE] Add an 'about' section

### ISSUE THREAD

**Issue by e-minguez** [mrmartineau/gatsby-theme-code-notes#47]
It would be nice to have an 'about'/'more info' section where you can explain what is the purpose of the site, authors, etc. without 'about' being a tag.

**Comment by mrmartineau** [comment]
@e-minguez tags are optional so you could just have an `about.md` and use that. But I think I know what you are thinking of, something more "official" that has it's own place in the sidebar perhaps and isn't in the list of notes. Is that it?

**Comment by e-minguez** [comment]
> @e-minguez tags are optional so you could just have an `about.md` and use that. But I think I know what you are thinking of, something more "official" that has it's own place in the sidebar perhaps and isn't in the list of notes. Is that it?

Exactly, something that can be considered 'first class citizen' :D (I was thinking an 'about' button just below the logo?)
